### DISCUSSION TITLE
Is this possible to make, moving or showing off objects ins GearsBot?

### DISCUSSION THREAD

**Discussion by PatrykKoro01** [QuirkyCort/gears#102] [Ideas]
I want my box to show up after few seconds after starting program. Is this possible?

**Comment by QuirkyCort** [comment]
In a custom world? No. But you can place the box somewhere out of the way and make it move into location after a few seconds. For that, you can use the animation feature.

Here's an example if you want the box to appear after 5 seconds.

1) Add your box. Move it out of the way.
2) Set "animation mode" to either "Loop" or "Alternate".
3) Add an animation key.
4) Set the time to shortly before 5 seconds (eg. 4.9 seconds). Add a second animation key.
5) Set the time to 5 seconds. Add a third animation key.
6) Set the time to a very large value that you will not reach (eg. 6000 seconds). Add the fourth and last animation key.

**Reply by PatrykKoro01** [reply]
Thanks. I have more questions. I am doing some kind of Mars Rover. It will shot aliens with paintball launcher. Is this possible that alien will disappear after got shot? And is there any command in Python that will make my rover go to next alien after shot? (like touch_sensor.pressed)? I can send you what have I done so far.

**Reply by PatrykKoro01** [reply]
And there is any option that this box will stop in position that i want?

**Comment by QuirkyCort** [comment]
Not in custom world, but you can create your own world in Javascript. With that, you can do whatever you want, including making the box appear after some time and disappear on paintball hit.

If you make the box disappear after being hit, you might be able to detect that with a laser range sensor. You can then write a robot program that detects that the box is no longer there and move on to the next target.
